sdb/commands/pretty_print.py

```diff
@@ -23,6 +23,9 @@
 
 
 class PrettyPrint(sdb.Command):
+    """
+    TODO
+    """
```

Comment:
Is it useful to have this here, as opposed to not having a specific help message?

Reply:
Sorry, this was meant as a indication I need help :)
It appears that most things do a 'pretty print' by default. When would you need to call this command directly?

Reply:
I'd be OK leaving this undocumented for now, since as you say, most things will pretty print by default.

Reply:
Removed